lib/datadog/tracing/contrib/roda/integration.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# typed: false
+
+require_relative '../integration'
+require_relative 'configuration/settings'
+require_relative 'patcher'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Roda
+        # Description of Roda integration
+        class Integration
+          include Contrib::Integration
+
+          MINIMUM_VERSION = Gem::Version.new('2.0.0')
+          MAXIMUM_VERSION = Gem::Version.new('4.0.0')
```

Comment:
Good to know this maximum. Do we know we don't support Roda 4.0.0? If so, why don't we?

Also "maximum" version as a name is slightly misleading in that its exclusive (we don't actually support 4.0.0.)

Reply:
At the time of this comment: Roda 4.x isn't out. The latest version is 3.65.0. 

Given the changes that occurred between 1.x and 2.x, I'm just being cautious by setting a maximum of 4.x for now. That way, in our public docs, we can be clear that this logic is intended for 2.x and 3.x. 

Whenever 4.x comes out, it'll require visiting to make sure it still functions as expected.